src/clicache/integration-test/ThinClientFunctionExecutionTestsN.cs

```diff
@@ -959,7 +959,7 @@ public void ExecuteFETimeOut()
         region[i] = "VALUE--" + i;
       }
 
-      object args = 5000;
+      object args = 5000 * 1000;
```

Comment:
I felt it was important to call out the disconnect between units here rather than hide it in a constant.